code/android-core-library/src/main/java/com/adobe/marketing/mobile/Event.java

```diff
@@ -326,7 +326,7 @@ void setResponseID(String responseID) {
 	/**
 	 * @return event timestamp in seconds
 	 */
-	long getTimestampInSeconds() {
+	public long getTimestampInSeconds() {
```

Comment:
@praveek let me know if you are ok with making this public. If not, can also manually convert to seconds from `getTimestamp`